configs/development.py

```diff
@@ -6,6 +6,7 @@
 SECRET_KEY = 'e951e5a1f4b94151b360f47edf596dd2'
 BIND_ADDRESS = '0.0.0.0'
 PORT = 9191
+OFFLINE_MODE = True
```

Comment:
I think OFFLINE_MODE should have the False value by default. So we can keep the original state for the existing PDA users when they do upgrading.

Reply:
`OFFLINE_MODE` should be available in the `default_config.py` file as well.